src/ResourceManager/EventHub/ChangeLog.md

```diff
@@ -18,9 +18,23 @@
         - Additional information about change #1
 -->
 ## Current Release
-* Added Location Completer to -Location parameters allowing tab completion through valid Locations
```

Comment:
This shouldn't be deleted

Reply:
Actually, this now needs to be resolved as we have merged back to preview. Please ensure that you simply are adding your blurb to the "Current Release" section without deleting anything.